examples/siro_sandbox/server/multiprocessing_config.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+# multiprocessing.dummy uses multithreading instead of multiprocessing
+use_dummy = False
```

Comment:
@0mdc beware I've been using `use_dummy = True` up until today. I just pushed a fix for "real" multiprocessing and I'm suggesting we default to real multiprocessing, to help server runtime perf. However, you should always feel free to switch to `use_dummy = False` when troubleshooting multithreading/networking issues.